docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -0,0 +1,134 @@
+
+The most common integration pattern with Digital Twins aims to offer asset-related data in expectable ways along the
+supply chain. This scenario assumes that the supplier offers data about an asset that was sold to an OEM further up the
+chain. Examples for this may include Bills of Material, carbon footprints, technical specifications or regulatory 
+documents. An OEM - or really any interested and privileged third party - can assume that a part's supplier offers a Digital
+Twin if the data-exchange is based on the Digital Twin Kit (and by proxy CX-0002). The data provider deploys at least:
+- Digital Twin Registry (DTR)
+  - registered in a DSP-conformant connector offering the DTR to the Dataspace.
+  - registers Digital Twins (ShellDescriptors in AAS vocabulary) with specificAssetIds including at least one that's known
+  at supplier and customer.
+- Submodels
+    - registered in a DSP-conformant connector to the Dataspace.
+    - registered to the correct ShellDescriptor in the DTR.
+    - Serving arbitrary but well-specified json from the `/$value` endpoint
+
+The Data Consumer's only responsibility is to deploy a connector and fetch the data from the right offers.
+
+A Bill of Material ([like this one](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.single_level_bom_as_built/2.0.0/gen/SingleLevelBomAsBuilt.json))
+for instance returns a list of `childItems` that hold for each subcomponent the information that's assumed to be present
+for the start of this very scenario (see Assumptions 1, 2). Thus, BoM and the DT deployment in combination 
+enable recursive browsing of the parts-tree, collecting data along the way.
+
+### 2. Adding a new Twin for a given Asset
+
+| Scenario                                                                                                      | Participants                                                   | Assumptions                             | Links to Use-Cases    |
+|---------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|-----------------------------------------|-----------------------|
+| A party (who is not the supplier of an part) publishes data about a previously uncovered aspect of the asset. | - Data Provider (Supplier) <br/> - Data Provider (Third Party) | 1. Third Party knows an id of the asset | - PURIS (coming soon) |
+
+Let's suppose, a car has reached it end-of-life. The object is sold to a dismantler who disassembles the vehicle and wants
+to publish that exact information. As the Digital Twin is owned by the OEM of the vehicle, it will not be accessible to
```

Comment:
The owner decides who is allowed to register new submodels to a digital twin. Typically it is the owner him or herself who is registering submodels only. Additionally, there might be legal or architectural reasons to copy/past submodels from another party. This is why it is possible that one and the same asset (identified via its global asset ID) may have more than one digital twin.